pytorch_lightning/trainer/trainer.py

```diff
@@ -569,9 +569,9 @@ def __init__(
         # configure profiler
         self.__init_profiler(profiler)
 
-        # init logger flags
+        # configure logger flags
         self.logger: Optional[LightningLoggerBase]
```

Comment:
I played a bit with this and this definition needs to be in the `__init__`, otherwise mypy will treat it as `Any`